### PR TITLE
Add logos and favicon

### PR DESCRIPTION
**Pull request by dmohs**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -7,7 +7,11 @@ import { Component } from '@angular/core';
   selector: 'aou-app',
   styleUrls: [ './app.component.css' ],
   template: `
-    <h1>{{title}}</h1>
+    <div style="display: flex; align-items: flex-end;">
+      <img src="images/all-of-us-logo.svg" alt="All of Us Logo" style="width: 18rem;">
+      <img src="images/portal-logo.svg" alt="Researcher Portal Logo"
+        style="margin-left: 0.5rem; width: 13rem;">
+    </div>
```
Comment (dmohs) [inline src/app/app.component.ts]: I don't like the pattern here of mixing inline styles with a CSS file, but it also seems super overkill to have to think of a class name to assign to each element. Is there an alternative?

Comment (markfickett) [reply src/app/app.component.ts]: I agree that I don't know a great solution. My preference is keeping all the styles in a separate stylesheet, even if some of them are selecting by ID, just so they're all together.

File: src/app/app.component.ts [hunk 2]
```diff
@@ -7,7 +7,11 @@ import { Component } from '@angular/core';
   selector: 'aou-app',
   styleUrls: [ './app.component.css' ],
   template: `
-    <h1>{{title}}</h1>
+    <div style="display: flex; align-items: flex-end;">
+      <img src="images/all-of-us-logo.svg" alt="All of Us Logo" style="width: 18rem;">
```
Comment (markfickett) [inline src/app/app.component.ts]: How about {{title}} as the alt?

My approach is to use the alt text not necessarily as a description of the image, but as a replacement for it -- so in this case, I assume the SVG has text similar to {{title}}, and using {{title}} in its place for screen readers would be a reasonable substitution.

File: src/images/all-of-us-logo.svg [hunk 3]
```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (markfickett) [inline src/images/all-of-us-logo.svg:1]: It looks like SVGs are pretty widely supported these days, nice. http://caniuse.com/#feat=svg Do you know if there's a way we can find expected browser/device usage? I wonder if it's very different from in general on the web. Also, are there any contractual constraints on browser support (hopefully not)?

I don't know if we should care about file size much, or whether a compressed/crushed PNG would be smaller than this (especially if we have to produce multiple sizes for different devices).

File: src/app/app.component.css [hunk 4]
```diff
@@ -1,7 +1,5 @@
-h1 {
-  font-size: 1.2em;
-  color: #999;
-  margin-bottom: 0;
+nav {
+  margin-top: 1rem;
```
Comment (markfickett) [inline src/app/app.component.css]: I hadn't seen "rem" units before, cool. That seems potentially more sane than specifying sizes relative to whatever the inherited/parent's style happens to be.

File: src/app/app.component.ts (hunk 1 above)
Comment (dmohs) [reply src/app/app.component.ts]: I don't think IDs buy us anything over class names—I'd still have to invent one for each element to which I want to apply style. The additional consideration with IDs is that they should be globally-unique, so we wouldn't be able to use them on components that are expected to be used more than once on a page.

File: src/images/all-of-us-logo.svg (hunk 3 above)
Comment (dmohs) [reply src/images/all-of-us-logo.svg:1]: Great question. I'll see if I can dig up an answer.

File: src/app/app.component.ts (hunk 1 above)
Comment (markfickett) [reply src/app/app.component.ts]: Oh, good point. Class names (which Angular helpfully namespaces for us) SGTM.

**Review by markfickett — APPROVED** [review]

File: src/images/all-of-us-logo.svg (hunk 3 above)
Comment (dmohs) [reply src/images/all-of-us-logo.svg:1]: FYI: I asked David Siedzik to track this down. We may need to follow-up later.

**Pull request merged**